cargo-workspaces/Cargo.toml

```diff
@@ -43,6 +43,8 @@ tame-index = { version = "0.9.0", features = ["git", "sparse"] }
 dunce = "1.0.4"
 ctrlc = "3.4.1"
 toml_edit = "0.19.10"
+cargo-util-schemas = "0.1.0"
+toml = "0.8.8"
```

Comment:
We are already bringing in `toml_edit`, why not use that?

Reply:
The main reason I chose to ho this route is that `cargo-util-schemas` fully describes Rusts Cargo.toml files as a Rust type. whereaOn the other hand the `toml_edit::Document` only describes a general toml file in a very weakly typed way (you never know if a value is an array, table, item etc unless you cast it)

The benefits of using the new stuff:

- no need to cast
- direct access to all allowed keywords in Cargo.toml-s

Issues with the new stuff:

- formatting is either all inline or pretty print (a.k.a. one item per line)

I guess you're right in that we should use the current approach and throw away my addition since the formatting is a main issue :/ I just wanted to try something new. Will change it :+1:

Reply:
I un-did this and used `toml_edit` instead for the initial formatting :+1: (also added a info log for the users which crates are automatically added to the workspace)